docs/reference/module-options.md

```diff
@@ -136,4 +138,252 @@ You might use that value as a fallback for user-editable fields.
+
+Option settings in this section apply to all piece modules (those that extend `@apostrophecms/piece-type`).
+
+- [`pluralLabel`](#plurallabel)
+- [`perPage`](#perpage)
+- [`publicApiProjection`](#publicapiprojection)
+- [`quickCreate`](#quickcreate)
+- [`searchable`](#searchable)
+
+### `pluralLabel`
+
+Similar to `label` for all doc types, the `pluraLabel` option sets the string the user interface will use to describe a piece type in plural contexts. All page types are referred to as "Pages" in these contexts, but pieces have unique labels, for example, in the manager modal where it might display "All Articles."
+
+If no `pluralLabel` value is provided, Apostrophe will append the `label` (whether set manually or generated [as described](#label)), with "s", as is typical for English words. **Even in English this is often not correct, so `pluralLabel` should usually be defined explicitly.**
+
+#### Example
+
+```javascript
+// modules/goose/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    label: 'Goose',
+    pluralLabel: 'Geese'
+  },
+  // ...
+}
+```
+
+### `perPage`
+
+In piece type modules, the `perPage` option, set to an integer, sets the number of pieces that will be returned in each "page" [during `GET` requests](api/pieces.md#get-api-v1-piece-name) that don't specify an `_id`. This value defaults to 10.
+
+#### Example
+
+```javascript
+// modules/article/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    perPage: 20 // REST `GET` requests will return 20 pieces per page.
+  },
+  // ...
+}
+```
+
+### `publicApiProjection`
+
+By default the built-in Apostrophe REST APIs are not accessible without proper [authentication](/reference/api/authentication.md). You can set an exception to this for `GET` requests to return specific document properties with the `publicApiProjection` option.
+
+This should be set to an object containing individual field name keys set to `1` for their values. Those fields names included in the `publicApiProjection` object will be returned when the `GET` API requests are made without authentication.
+
+#### Example
+
+```javascript
+// modules/article/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-type',
+  options: {
+    publicApiProjection: {
+      title: 1,
+      authorName: 1,
+      _url: 1 // 👈 Dynamic properties are allowed
+    }
+  },
+  // ...
+}
+```
+
+Unauthenticated [`GET /api/v1/article`](api/pieces.md#get-api-v1-piece-name) requests would return each piece with only the title, `authorName`, and `_url` properties.
+
+### `quickCreate`
+
+Setting `quickCreate` to the boolean `true` on a piece adds that piece type to the admin bar "quick create" menu. The Apostrophe admin bar user interface includes the quick create menu button to add new pieces without first opening their respective manager modals. This is a useful setting for the piece types that editors will be adding most often.
```

Comment:
"to the boolean" `true`" Seems like a clunky way to say this. Wouldn't this be better expressed as an option|type|value table?

"This is a useful setting for the piece types that editors will be adding most often." Not necessary. We wax-on sometimes in ways I think makes it a bit more difficult to scan.

Reply:
A table might be useful for some of this. We could do it like the REST API pages with each section getting a leading table that links to the deeper descriptions. I'm thinking columns for option, value type (e.g., boolean), and brief description.

Maybe a good thing to do. This is where we could use some designer thinking. I'd also do that in a new PR if we're going to do it.